doc/sdk_with_carto.md

```diff
@@ -2,6 +2,40 @@
 
+## Loading CARTO Map Data to Mobile Apps
+
+You can also manage mobile rendering by defining how mobile features load with the CARTO APIs and with the CARTO Builder. For example:
+
+- To use **raster map tiles**, define the tile URL for RasterTileLayer
+
+- To apply **interactivity** (object click data), use UTFGrid. This uses both raster map tiles and json-based UTF tiles
+
+  **Tip:** For CARTO Builder, you will need to enable and define tooltips with the [POP-UP](/docs/carto-builder/map-layers-for-rendering-data/#pop-up) feature
+
+- Load **vector tiles**, the CARTO Engine supports Mapbox Vector Tile (MVT) format tiles, which the Mobile SDK can render on the client side. You will also need [CartoCSS](https://carto.com/docs/carto-engine/cartocss/) styles to view vector tiles. This is useful for applying advanced styling features, such as zooming and rotating maps based on data that can be packaged for offline line, using mbtiles
+
+- **Load GeoJSON vector data**. This is useful if you need need more advanced interactivity (object click actions) or dynamic client-side styling of the objects. For vector data, the CARTO Engine provides a [SQL API](/docs/carto-engine/sql-api/) and mobile app that can load entire tables and render maps. You can also use client-side simplification and clustering
+
+- If the **data table is large** (more than a few thousand objects), then loading entire tables can overload the mobile client. Alternatively, use on-demand, view-based loading of vector data. Similar to the SQL API and GeoJSON format used on the CARTO Engine side, the SDK applies custom vector data sources to load data. _Only a selected, visible area, of the map will load._ The mobile app can control zoom levels, server-side generalizations, and simplifications can be applied
+
+- For point-geometry time-series visualizations, use the _Animated_ aggregation to define Torque maps. This provides animated rendering, and the Mobile SDK has a special layer `TorqueTileLayer` to define this. From an API standpoint, Torque uses the SQL API and CartoCSS styling, but Torque contains an additional [time control method](/docs/carto-engine/torque-js/)
+
+## Publish a Mobile Map
+
+This high-level workflow describes how to prepare your mobile data for rendering, using the Import API and CARTO Builder.
+
+1) Upload your map data using the [Import API](/docs/carto-engine/import-api/)
+
+2) Create a map with the [CARTO Builder](https://carto.com/learn/guides)
+
+3) Click _SHARE_ to publish your map
+
+4) Select _CARTO Mobile SDK_ as the publish option
+
+  This publishing option provides a URL to the mobile viz.json file, which is required if you are using the CARTO Mobile SDK to publish custom maps for Android, iOS, and Windows platforms.
+
+  **Note:** [Widgets](https://carto.com/learn/guides/widgets/exploring-widgets) are currently not supported for the CARTO Mobile SDK format, but will be available in a future release.
+
 ## SDK and CARTO APIs
 
```

Comment:
@jaakla , I moved this section down. Is that what you wanted to do? 
Please ensure that this is not dup content of line #639.